fhir-server-spi/src/main/java/com/ibm/fhir/server/spi/operation/FHIRResourceHelpers.java

```diff
@@ -76,7 +76,8 @@ public static Interaction from(String value) {
      * @param interaction
      *            the interaction to be performed
      * @param resourceType
-     *            the resource type against which the interaction is to be performed
+     *            the resource type against which the interaction is to be performed; use "Resource"
+     *            for whole-system interactions
```

Comment:
this is the only intentional change in this file...the rest is just whitespace cleanup